views/xibo-audience-connector-form-settings.twig

```diff
@@ -27,7 +27,13 @@
 {% block callBack %}audienceFormOpen{% endblock %}
 {% block connectorFormFields %}
 
-    <h4>Coming soon</h4>
-    <p>We're working on bringing you this exciting connector in the next update.</p>
+    {% if not interface.isProviderSetting("apiKey") %}
+    <h4>Settings</h4>
+    <p>Your API key allows for secure communication between the CMS and the Xibo audience service. It is used
```

Comment:
I think we can translate this message, and update it to reflect what it does (analyse your proof of play data for Ad Campaigns and retrieve reports).